axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveCounterPartServiceImpl.java

```diff
@@ -110,7 +118,8 @@ else if (move.getJournal()
           .getJournalType()
           .getTechnicalTypeSelect()
           .equals(JournalTypeRepository.TECHNICAL_TYPE_SELECT_EXPENSE))
-        accountingAccount = move.getCompany().getAccountConfig().getSupplierAccount();
+        accountingAccount =
+            accountConfigService.getAccountConfig(move.getCompany()).getSupplierAccount();
```

Comment:
There is a specific class to get the fields of the AccountConfig object, exemple : https://github.com/axelor/axelor-open-suite/blob/master/axelor-account/src/main/java/com/axelor/apps/account/service/config/AccountConfigService.java#L290